app/views/annual_evaluations/_form.html.slim

```diff
@@ -1,33 +1,44 @@
 = wrapped_form_for([@patient, @annual_evaluation]) do |f|
   = f.error_notification
-
   .form-inputs
-    h2 Clinical
-    = f.input :bmi
-    = f.input :bladder_drainage_method, collection: AnnualEvaluation.bladder_drainage_methods.keys
+    h4 Clinical
+    = f.input :bmi, label: "BMI", :placeholder => "00"
+    = f.input :bladder_drainage_method,  label: "Bladder Drainage Method", :prompt => "Please select one", collection: AnnualEvaluation.bladder_drainage_methods.keys
     = f.input :has_motor_or_sensory_asymmetry
-    = f.input :neuro_zone_of_preservation1, collection: AnnualEvaluation.neuro_zone_of_preservation1s.keys
-    = f.input :neuro_zone_of_preservation2, collection: AnnualEvaluation.neuro_zone_of_preservation2s.keys
-    = f.input :neuro_zone_of_preservation3
+    = f.input :neuro_zone_of_preservation1,  label: "Neuro Zone of Preservation", :prompt => "Please select one", collection: AnnualEvaluation.neuro_zone_of_preservation1s.keys
+    = f.input :neuro_zone_of_preservation2, label: false, :prompt => "Please select one", collection: AnnualEvaluation.neuro_zone_of_preservation2s.keys
+    = f.input :neuro_zone_of_preservation3, label: false, :placeholder => "00"
 
-    h2 ASIA and LOI
-    = f.input :level_of_injury
-    = f.input :asia_score
+    h4 ASIA &mdash; LOI
+    .instrument
+        h4
+          | ASIA-LOI: A-C6  
+          span.small = link_to "view record: 12/23/1987"
```

Comment:
The link also doesn't link to anything! 

I will add a TODO for all the things it needs to eventually do